### DISCUSSION TITLE
account lost

### DISCUSSION THREAD

**Discussion by josisayu** [community/community#41961] [Other Feature Feedback, Questions, & Ideas]
i have an account that named Jo2831 and i use to it for the past above 8 month and i have many repo in it  now my pc history was remove then when i try to login to my GitHub account,
it redirect to my old GitHub account that i created for test purpose when i try to learn how to create GitHub account(this one), and now i can't access in my active GitHub account please help me i need my account that have many document.

**Comment by radar-blip** [comment]
For account related issues, see here: [Support page](https://support.github.com/contact/account)
Give all details you can to make it easier for support to help you.

**Comment by ck12330** [comment]
If you cannot access new version you should have a backup folder to drive or cloud even it has been deleted from computer uninstall the app and download the new version and restore the from your drive or cloud or go to your recybin/trash option to restore it

**Reply by ck12330** [reply]
You also get the new version available link (This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 2.0, available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.)

**Comment by mohamedzhioua** [comment]
when you have this kind of issue you should try the support page [Support page](https://support.github.com/contact/account)
